bindings/wasm/src/tangle/client.rs

```diff
@@ -41,34 +42,22 @@ pub struct WasmClient {
 
 #[wasm_bindgen(js_class = Client)]
 impl WasmClient {
-  fn from_client(client: Client) -> WasmClient {
-    Self {
-      client: Rc::new(client),
-    }
-  }
-
   /// Creates a new `Client` with default settings.
   #[wasm_bindgen(constructor)]
   pub fn new() -> Result<WasmClient> {
-    Self::from_config(&mut Config::new())
+    // TODO: is there a way to avoid blocking? Async constructors are not supported.
```

Comment:
In Napi we wrote a static method to avoid the `block_on` call. Not sure if we want this here though

Reply:
The places where this pattern occurs are legacy code from a while back; I'm not sure if we want to replace them with a static method everywhere given the convenience/expectation of constructors in our public API though.

Reply:
Well, async constructors are possible in JS, but it's not recommended to use them since it will confuse users. Also in JS the constructor, is not supposed to do anything other than make an instance of a class. Anything else should be done in a separate method. we can do something like
```JavaScript
let client = new Client();
await client.init();
```
But this doesn't look great either. 

I personally don't think that a static method is a bad idea. But if the blocking only needs a constant time, it's not a problem IMO.

Reply:
I'm of the opinion that Javascript is wrong here, building the class should make it immediately usable.

Looking at the code it doesn't seem like the `ClientBuilder.build/finish` method actually awaits anything (except in some dead, commented-out code) so blocking should be fine for now.